Jetcaster/core/src/main/java/com/example/jetcaster/core/player/MockEpisodePlayer.kt

```diff
@@ -129,6 +133,35 @@ class MockEpisodePlayer(
         next()
     }
 
+    override fun play(playerEpisodes: List<PlayerEpisode>) {
```

Comment:
This looks almost identical to the logic in of `play(playerEpisode: PlayerEpisode)`. How about modifying that function to call this function in its implementation?
i.e.

```
override fun play(playerEpisode: PlayerEpisode) {
    play(listOf(playerEpisode))
}
```

Reply:
good point! done